x-pack/legacy/plugins/lens/public/indexpattern_plugin/operation_definitions/terms.tsx

```diff
@@ -42,11 +43,15 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn> = {
     defaultMessage: 'Top Values',
   }),
   getPossibleOperationsForDocument: () => [],
-  getPossibleOperationsForField: ({ aggregationRestrictions, type }) => {
-    if (type === 'string' && (!aggregationRestrictions || aggregationRestrictions.terms)) {
+  getPossibleOperationsForField: ({ aggregationRestrictions, aggregatable, type }) => {
+    if (
+      (type === 'string' || type === 'boolean') &&
```

Comment:
Are we going to treat IP addresses as strings as well? I think we want Terms support for those

Reply:
Right, I will add this in a separate PR.